src/typescript.js

```diff
@@ -1,14 +1,17 @@
 
 const { Module } = require('module');
+
 const m = new Module('', null);
+const log = (...args) => { if (!quiet) { console.log(...args); } }
```

Comment:
This function isn't used?

Reply:
good catch, i'll remove it in next commit.